com.unity.robotics.urdf-importer/Runtime/Extensions/UrdfRobotExtensions.cs

```diff
@@ -241,7 +241,10 @@ public static void CorrectAxis(GameObject robot)
 
             foreach (UrdfCollision collision in collisionMeshList)
             {
-                collision.transform.localRotation = collision.transform.localRotation * correction;
+                if (robotScript.choosenAxis != ImportSettings.axisType.zAxis) 
```

Comment:
is `robotScript` guarranteed to be not null or do we need a null check?

Reply:
Good point! This is a public class so we probably need to have the null check. Just updated and added the null check to the beginning of the method.